docarray/array/storage/weaviate/backend.py

```diff
@@ -117,6 +144,14 @@ def _init_storage(
             f'{config.protocol}://{config.host}:{config.port}',
             timeout_config=config.timeout_config,
         )
+        self._client.batch(
```

Comment:
what does this batch do?

Reply:
This is the Weaviate client configuration when choosing Weaviate as a storage engine (hence it's in `_init_storage()`)

I've added additional information in the code itself.